src/Diagnostics.ModelsAndUtils/Utilities/AzureSupportCenterInsightUtilities.cs

```diff
@@ -123,6 +124,52 @@ public static AzureSupportCenterInsight CreateDefaultInsight<TResource>(Operatio
             return supportCenterInsight;
         }
 
+        /// <summary>
+        /// Use this when the site is not found from observer
+        /// </summary>
+        public static AzureSupportCenterInsight CreateSiteNotFoundInsight(string subscriptionId, string resourceGroupName, string siteName)
+        {
+            string kustoWebLink = $"https://dataexplorer.azure.com/clusters/wawswus.kusto.windows.net/databases/wawsprod?" +
```

Comment:
This Kusto link connects to wawswus. If the site is any other region, the Kusto link will fail.

Reply:
I see, what will be the solution for that? I used All() function in query, but I am not sure which kusto server should I choose to run the query.

Reply:
My bad @yifguoMSFT , I didn't notice the "All" in your query earlier. This'll work.